### PR TITLE
Fix #5959: dependencyBrowseGraph fails with Couldn't load 'graph.html'

### LINKED ISSUE #5959: dependencyBrowseGraph task fails with sbt 1.4.0
## steps

1. Generate new project using scala seed
```
sbt new scala/scala-seed.g8
```
2. Update `project\build.properties` to use sbt 1.4.0
```
sbt.version=1.4.0
```
3. Create file `project\plugins.sbt` with follwing content
```
addDependencyTreePlugin
```
4.  run following command
```
sbt dependencyBrowseGraph
```

## problem

sbt fails with following stack trace

```
PS C:\Dev\Projects\dwh\bla\scala-seed-project> sbt dependencyBrowseGraph
[info] welcome to sbt 1.4.0 (AdoptOpenJDK Java 1.8.0_265)
[info] loading settings for project global-plugins from updates.sbt ...
[info] loading global plugins from C:\Users\id967052\.sbt\1.0\plugins
[info] loading settings for project scala-seed-project-build from plugins.sbt ...
[info] loading project definition from C:\Dev\Projects\dwh\bla\scala-seed-project\project
[info] loading settings for project root from build.sbt ...
[info] set current project to Scala Seed Project (in build file:/C:/Dev/Projects/dwh/bla/scala-seed-project/)
[error] java.lang.IllegalArgumentException: requirement failed: Couldn't load 'graph.html' from classpath.
[error]         at scala.Predef$.require(Predef.scala:281)
[error]         at sbt.internal.graph.rendering.TreeView$.saveResource(TreeView.scala:57)
[error]         at sbt.internal.graph.rendering.DagreHTML$.createLink(DagreHTML.scala:22)
[error]         at sbt.plugins.DependencyTreeSettings$.$anonfun$browseGraphHTMLTask$1(DependencyTreeSettings.scala:186)
[error]         at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error]         at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error]         at sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error]         at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error]         at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error]         at sbt.Execute.work(Execute.scala:291)
[error]         at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error]         at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error]         at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error]         at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error]         at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error]         at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error]         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error]         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error]         at java.lang.Thread.run(Thread.java:748)
[error] (Compile / dependencyBrowseGraphHTML) java.lang.IllegalArgumentException: requirement failed: Couldn't load 'graph.html' from classpath.
[error] Total time: 0 s, completed 08-Oct-2020 18:00:31
```

## expectation

Should generate dependency tree HTML file and open Browser with it.

### PR DESCRIPTION
**Pull request by naderghanbari**
- Add missing graph.html and tree.html resources from
https://github.com/sbt/sbt-dependency-graph
- Fixes `dependencyBrowseTree`, `dependencyBrowseTreeHTML`,  `dependencyBrowseGraph`,
and `dependencyBrowseGraphHTML` tasks

### CODE REVIEW

**Comment by naderghanbari** [comment]
Fixes #5965

**Comment by naderghanbari** [comment]
Fixes #5959

File: main/src/main/resources/graph.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+THE SOFTWARE.
+
+-->
+
+<meta charset="utf-8">
+<title>Dependency Graph</title>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.17/d3.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+<script src="https://www.samsarin.com/project/graphlib-dot/v0.6.1/graphlib-dot.js"></script>
```
Comment (jtjeferreira) [inline main/src/main/resources/graph.html]: Maybe this a good time to stop using this samsarin.com domain...

Comment (naderghanbari) [reply main/src/main/resources/graph.html]: Sure, I added a commit. Unfortunately `graphlib-dot` is not actively maintained, so the best I could find, according to [its own wiki](https://github.com/dagrejs/graphlib-dot/wiki#browser-scripts) was to use GH cdn. Version `0.6.1` was not available despite being marked as the latest release in the [release page](https://github.com/dagrejs/graphlib-dot/releases) but version `0.6.3` which seems to be the latest version was. I tested the HTML page after this upgrade and it works fine.

There are other alternatives (now in 2020) so maybe in long term switching to other l`dot` libraries is a good move.

**Review by eed3si9n — APPROVED** [review]
Thanks!

**Pull request merged**